src/components/Pages/Admin/Designer.vue

```diff
@@ -466,8 +338,18 @@ export default {
             const xml = this.GI.getXML();
             const metaInformation = this.produceXMLWithoutConstructionInside(xml);
             const perspectives = this.extractPerspectives(metaInformation);
+            let toolbar = null;
```

Comment:
The code in `send` method is duplicated in `sendToAll`. Please extract it to separate method.

This method should return either `null` if the toolbar should not be sent or a toolbar definition embedded in the request. This should allow you to avoid disabling `prefer-destructuring` in the linter.